core/jvm/src/main/scala/zio/internal/ZScheduler.scala

```diff
@@ -205,8 +208,9 @@ private final class ZScheduler extends Executor {
   private[this] def makeSupervisor(): ZScheduler.Supervisor =
     new ZScheduler.Supervisor {
       override def run(): Unit = {
-        var currentTime      = java.lang.System.currentTimeMillis()
-        val previousOpCounts = Array.fill(poolSize)(-1L)
+        var currentTime         = java.lang.System.currentTimeMillis()
```

Comment:
Out of curiosity, why not `nanoTime?`. I've always heard that measuring with `currentTimeMillis` is known to cause [issues](https://www.javaadvent.com/2019/12/measuring-time-from-java-to-kernel-and-back.html).